templates/rhdm710-authoring-ha.yaml

```diff
@@ -720,9 +720,7 @@ objects:
           - name: SERVICE_PROFILE
             value: ${APPLICATION_NAME}-datagrid
           - name: JGROUPS_PING_PROTOCOL
-            value: openshift.DNS_PING
-          - name: OPENSHIFT_DNS_PING_SERVICE_NAME
-            value: ${APPLICATION_NAME}-datagrid-ping
+            value: KUBERNETES_NAMESPACE
```

Comment:
looks like it is wrong.
plus like we discussed on the operator PR, don't change infinispan and amq configuration if its docs still point to openshift.PING.

Reply:
Fixed protocol, infinispan and amq conf is broken if we use JGROUPS_PING_PROTOCOL=kubernetes.KUBE_PING OPENSHIFT_DNS_PING_SERVICE_NAME and OPENSHIFT_DNS_PING_SERVICE_PORT are used if JGROUPS_PING_PROTOCOL=kubernetes.KUBE_PING is openshift.DNS_PING